src/foods/dto/create-food.dto.ts

```diff
@@ -19,4 +20,6 @@ export class CreateFoodDto {
 
   @IsNumber({ allowNaN: false, allowInfinity: false })
   category: Category;
+
+  createBy: User;
```

Comment:
User or userId?

Reply:
@tar-aldev in the DB will create column with name "createById" and this column will always return by request. but if we specify relation "createBy" then DB will return User object